0.3/Arcs/source/Launcher.js

```diff
@@ -112,13 +117,19 @@ ${style}
       return template;
     }
     _willReceiveProps(props) {
-      let items = [], profileItems = [];
+      let items = [],
+        profileItems = [];
       props.arcs.forEach((a, i) => {
         // each item goes in either the `items` or `profileItems` list
         let list = a.profile ? profileItems : items;
-        let blurb = a.description.length > 70 ? a.description.slice(0, 70) + '...' : a.description;
+        let blurb =
+          a.description.length > 70
+            ? a.description.slice(0, 70) + '...'
+            : a.description;
         list.push({
-          index: i,
+          arcId: a.id,
+          // Don't allow deleting the 'New Arc' arc.
+          disallowDelete: i == 0,
```

Comment:
Is the New Arc arc guaranteed to be first?

Reply:
It's pre-pended here: https://github.com/PolymerLabs/arcs-cdn/blob/gh-pages/dev/app-shell/elements/arc-app.html#L480

I guess it's treated as an "Arc" just because it was convenient, since in the current UI it's rendered in the same manner. The only parameterized value that we'd need to pass along is the user id, which the Launcher probably has, so we can see about reworking to not prepend it here, but still render it in the launcher. I don't think there would be any ramifications but maybe there's something I don't know about Arc-suggestion-wise.

We could also add a `deletable` boolean or similar to `ArcMetadata`, and set it to false for this one, but have it be true for other arcs. But since `New Arc` isn't really an arc, I lean toward removing it from the list of arcs instead.